index.html

```diff
@@ -41,11 +41,63 @@
         background: white;
         padding: 0.5em;
       }
+      .translateme::before {
+        content: ' Translate this: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
+      .translateme {
+        background-color: #F3F3C3;
+      }
+      .checkme {
+        background-color: #BBEFFC;
+      }
+      .checkme::before {
+        content: ' Check this translation: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
     </style>
   </head>
 <body>
 <section id="abstract">
-  <p>A simple set of rules for placement of Ruby text in Japanese typography.</p>
+  <p class="checkme">This document describes simple method of processing for Ruby 
+  in Japanese layout realized with technologies like CSS, SVG, and XML-FO, 
+  as references to rendering engine implementors. 
+  Unlike 
+  <a href="https://www.w3.org/TR/jlreq/"><abbr title="Requirements for Japanese Text Layout">JLReq</abbr></a>
```

Comment:
I'm leaning towards leaving only one link of jlreq here (i.e., only use `[[JLREQ]]`), but have no strong opinion.

Reply:
I suppose this part is to have abbr tag for JLreq, so let me keep this as is.